pysat/_constellation.py

```diff
@@ -136,6 +277,62 @@ def bounds(self, value=None):
 
         return
 
+    @property
+    def date(self):
+        """Date for loaded data."""
+        if len(self.index) > 0:
+            return utils.time.filter_datetime_input(self.index[0])
```

Comment:
I double checked in pysat and it sets the date in the same way, using first index. At times I've thought maybe a midpoint of loaded data or something else may be better. Checking in to see if you have any additional thoughts on the matter. If not, consider it resolved.

Reply:
No, I think the date is fine as the first time.  Since load assumes by default you're loading 1+ days, having the date as the date of the first time is a good representation of the loaded data.